parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -398,6 +413,89 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+// Add bridge pallets (GPA)
+/// Add GRANDPA bridge pallet to track Polkadot relay chain on Kusama BridgeHub
+pub type BridgeGrandpaPolkadotInstance = pallet_bridge_grandpa::Instance1;
+impl pallet_bridge_grandpa::Config<BridgeGrandpaPolkadotInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type BridgedChain = bp_polkadot::Polkadot;
+	type MaxFreeMandatoryHeadersPerBlock = ConstU32<4>;
+	type HeadersToKeep = RelayChainHeadersToKeep;
+	type WeightInfo = weights::pallet_bridge_grandpa::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	pub const RelayChainHeadersToKeep: u32 = 1024;
+	pub const ParachainHeadsToKeep: u32 = 64;
```

Comment:
I don't have any rationale for that, but imo this constant looks rather small. `64` means that if someone submits **every** bridged parachain header, the oldest one will be pruned in `(12s * 64) / 60 ~ 13 minutes`. As I said, there are no any issues with that, but I'd increase it at least to `300` (meaning `1 hour` of all bridged parachain headers). And maybe change the other constant (`RelayChainHeadersToKeep`) accordingly to `600` (meaning `1 hour` of all bridged relay chain headers).